src/backend/db/database_manager.py

```diff
@@ -259,3 +296,128 @@ def check_customer_guid_exists(self, customer_guid):
         finally:
             session.close()
 
+    #Custom Fields Related Functions
+    def add_custom_field(self, customer_guid, field_name, field_type, required):
+        customer_db_name = self.get_customer_db(customer_guid)
+        session = DatabaseManager._session_factory()
+
+        # Define a set of allowed SQL data types
+        allowed_sql_types = {"VARCHAR(255)", "INT", "BOOLEAN", "DATE", "MEDIUMTEXT"}
```

Comment:
Make it allowed_custom_field_sql_types
Make it a class variable instead of a function variable

Reply:
updated sir